src/main/webapp/eform/eformFloatingToolbar/eform_floating_toolbar.js

```diff
@@ -827,20 +829,33 @@ function remoteSave() {
 	 * A counter-measure to ensure images that are set by Javascript methods are captured
 	 * when the form is saved or rendered into a pdf.
 	 */
+	function appendImageInputs() {
+		jQuery("form[method='POST'] img").each(function () {
```

Comment:
I'm not sure this is a good idea. Any form tag with the attribute "post" could contain any number of malicious images.  How is this risk being mitigated?